tests/install/test_distribution.pm

```diff
@@ -9,7 +9,10 @@ sub run {
     diag('initialize working copy of openSUSE tests distribution with correct user');
     assert_script_run('username=bernhard email=bernhard@susetest /usr/share/openqa/script/fetchneedles', 3600);
     save_screenshot;
-    assert_script_run('retry -s 30 -- zypper -n in os-autoinst-distri-opensuse-deps', 600);
+    # os-autoinst-distri-opensuse is changing quickly and we can not trust the
+    # implicit repo refresh of 'zypper -n in' to work as proven by failures
+    # despite retry.
```

Comment:
There is a default `repo.refresh.delay` in `/etc/zypp/zypp.conf` of 10 minutes so this might explain why the autorefresh is not always working and might be a better proof than "failures despite retry"